development/PersonOntology.owl

```diff
@@ -167,31 +167,6 @@
     </owl:ObjectProperty>
     
 
-
-    <!-- https://w3id.org/enigma#author -->
-
-    <owl:ObjectProperty rdf:about="https://w3id.org/enigma#author">
-        <rdfs:subPropertyOf rdf:resource="http://purl.org/dc/terms/creator"/>
-        <rdfs:domain rdf:resource="https://w3id.org/enigma#AcademicArticle"/>
-        <rdfs:range rdf:resource="https://w3id.org/enigma#Person"/>
-        <rdfs:comment xml:lang="en">Property to describe the author of a publication in the context of an ENIGMA project</rdfs:comment>
-        <rdfs:label xml:lang="en">author</rdfs:label>
-    </owl:ObjectProperty>
-    
-
-
-    <!-- https://w3id.org/enigma#forOrganization -->
-
-    <owl:ObjectProperty rdf:about="https://w3id.org/enigma#forOrganization">
```

Comment:
This property was requested at some point by Neda... Not sure if it's ok to drop it, even if not used.